src/main/java/org/jenkinsci/plugins/bitbucket/BitbucketBuildStatusHelper.java

```diff
@@ -74,12 +75,14 @@
             throw new Exception("Bitbucket build notifier only works with SCM");
         }
 
+        final Jenkins jenkins = Jenkins.getInstance();
+
         ScmAdapter scmAdapter;
         if (scm instanceof GitSCM) {
             scmAdapter = new GitScmAdapter((GitSCM) scm, build);
         } else if (scm instanceof MercurialSCM) {
             scmAdapter = new MercurialScmAdapter((MercurialSCM) scm, build);
-        } else if (scm instanceof MultiSCM) {
+        } else if (scm instanceof MultiSCM && jenkins != null && jenkins.getPlugin("multiple-scms") != null) {
```

Comment:
won't this cause a class not found exception because of the import?
have you tested this without the plugin installed?

I would just remove the adaptor, from what I can tell the multi scm plugin is quite broken